portfolio/src/main/webapp/style.css

```diff
@@ -389,6 +394,15 @@ table {
 }
 /*End charts*/
 
+/*Start map*/
```

Comment:
Add spaces at the beginning and end of the comment:
`/* Start map */`

Reply:
added spaces!